NitroxServer/Communication/Packets/Processors/CellVisibilityChangedProcessor.cs

```diff
@@ -14,6 +14,9 @@ class CellVisibilityChangedProcessor : AuthenticatedPacketProcessor<CellVisibili
         private readonly EntitySimulation entitySimulation;
         private readonly PlayerManager playerManager;
 
+        private static int simCount = 0;
```

Comment:
Were these for debugging?  Can we remove?

Reply:
Same for the `Count` variables in `cellEntities` / `ownershipChange`?